pkg/odo/cli/component/push.go

```diff
@@ -158,7 +201,7 @@ func NewCmdPush(name, fullName string) *cobra.Command {
 		Short:       "Push source code to a component",
 		Long:        `Push source code to a component.`,
 		Example:     fmt.Sprintf(pushCmdExample, fullName),
-		Args:        cobra.MaximumNArgs(1),
+		Args:        cobra.MaximumNArgs(2),
```

Comment:
Yeah, this has to be updated because we were not correctly retrieve the arguments. I believe it's because `odo component create` is also an alias to `odo create`, so we put the max args as 2.

The `Use.` has already been updated to show that [component name] is passed.

Reply:
I don't see the change to `Use` field?

Reply:
@cdrage Shouldn't we update the `Use` field?